src/Elasticsearch/Objects/Documents.php

```diff
@@ -132,16 +219,13 @@ public function delete() : array
     public static function getById(int $id) : self
     {
         $params = [
-            'index' => (new static($id, []))->getIndices(),
+            'index' => (new static())->setData($id, [])->getIndices(),
```

Comment:
@FredPeal now we dont need to do this right? since before we had to pass this values to the construct, now we can just call new static()->getIndices()